projects/transactions-journey/src/lib/services/transactions-journey-config.service.ts

```diff
@@ -1,6 +1,8 @@
 import { Injectable } from '@angular/core';
 
-@Injectable()
+@Injectable({
+  providedIn: 'root',
+})
 export class TransactionsJourneyConfiguration {
```

Comment:
Why this service was add to the root injector?

Reply:
I have updated for both of Transactions services to be provided only in the scope of module.